### PR TITLE
README Rework

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**
## About The Pull Request

Please check the **[Preview](https://github.com/ElectronicsArchiver/TerraGov-Marine-Corps)**.

- Converted raw links to referenced links
- General formatting / spacing / separation
- Refactored licensing information into table
- Added license badges
- Linked license material sources

CC licenses are of version 4.0, but since they were
designed as direct upgrades to 3.0, there shouldn't
be any issues.

CC Attribution -> CC BY ? 

## Why It's Good For The Game

No the game, but its repo.

Better overview of licensing notices.
Cleaner look, improved readbility.

### CODE REVIEW

**Comment by mrhugo13** [comment]
Since this isn't actually affecting anything game-related, then it's probably better to not have it as part of the changelog.

**Comment by ElectronicsArchiver** [comment]
Sure, I removed the changelog section~

**Review by BraveMole — CHANGES_REQUESTED** [review]
I understand less this than the current readme.

And you are changing licences ? 

Ok it looks pretty but simple MD is good enough fore a readme, especially if it contains 10 lines

**Comment by ElectronicsArchiver** [comment]
As I said, CC4 licenses graduate CC3 ones 
( no license changes, they just refined them and are now the common standard )

> Ok it looks pretty but simple MD is good enough fore a readme, especially if it contains 10 lines

You say that, but have you compared the **[Old]** & **[New]** versions?
The actual content part is much cleaner .. 

[New]: https://github.com/ElectronicsArchiver/TerraGov-Marine-Corps/blob/master/README.md?plain=1
[Old]: https://github.com/tgstation/TerraGov-Marine-Corps/blob/master/README.md?plain=1

**Comment by BraveMole** [comment]
Ok but sentences are clearer than your table IMO

**Comment by ElectronicsArchiver** [comment]
Well it's not about reading in this case is it?

If you wanted to check for licensing you'd be interesting 
in for example the type kind of material, in that case 
you'd have to simply check the first column.

If you wanted to check what licenses the repo 
contains you would check the last column.

I'm not sure how in this situation a written version 
would be more of use than a diagram?

**Comment by BraveMole** [comment]
image

I do not get what's going on here, despite already knowing what's written on original readme. I do not expect someone just coming to this repo to understand either.

**Comment by ElectronicsArchiver** [comment]
And I'm honestly not sure what you don't understand about it.

**What** material 
**From** where
is under which **License**

*On that note, what is `-this commit-` anyways?*
What commit was meant with that?
( In the original the next referenced thing was a PR )

**Comment by ElectronicsArchiver** [comment]
I could rename the table headers if you'd like to for example 
`Material Type`, `Licensing Material`, `Source` or something.

File: README.md [hunk 1]
```diff
@@ -1,32 +1,96 @@
...
+
+*Contributions, if not otherwise specified,* <br>
+*are licensed under the **[AGPL][License AGPL]** license.*
+
+<br>
+<br>
+
+What            | From                                 | License
+:--------------:|:------------------------------------:|:------------------------:
+`Artwork`       | *after -this commit-*                | [![Badge CC-BY-NC]][License CC-BY-NC]
```
Comment (TiviPlus) [inline README.md:35]: Please add that this means https://github.com/tgstation/TerraGov-Marine-Corps/commit/92ee94da6e144d4420c1ec4f83a4ee785d61dc9b

File: README.md [hunk 2]
```diff
@@ -1,32 +1,96 @@
...
+<div align = center>
+
+*Contributions, if not otherwise specified,* <br>
+*are licensed under the **[AGPL][License AGPL]** license.*
+
+<br>
+<br>
+
+What            | From                                 | License
+:--------------:|:------------------------------------:|:------------------------:
```
Comment (TiviPlus) [inline README.md:34]: See discussion with brave maybe change this "What" to "Asset type" or similar?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**